Makefile

```diff
@@ -99,13 +106,6 @@ generate: bootstrap
 	./hack/tools/bin/cue export ./schemas/manifest.cue > ./parts/linux/cloud-init/artifacts/manifest.json
```

Comment:
should we make a `validate-manifest` as well?

personally I actually think we can remove all references to manifest.json from everywhere _except_ the provisioning scripts themselves, since we no longer actually use it when building VHDs. though we'll need to keep using it as a fallback in the provisioning scripts for cases where newer CSE/customdata is running on older VHDs